### ISSUE TITLE
Update to 10.0.8 failed; Possible Postgres 10 Doctrine regression

### ISSUE THREAD

**Issue by rasilon** [owncloud/core#31352]
Note that the error message is a lot like older Doctrine issues with PG10 that had been resolved recently, so I suspect some form or regression.  

### Steps to reproduce
1.  Upgrade from 10.0.7 to 10.0.8 with a Postgres 10 backend
2.
3.

### Expected behaviour
Upgrade should complete

### Actual behaviour
upgrade fails with this error:
Doctrine\DBAL\Exception\InvalidFieldNameException: An exception occurred while executing 'SELECT min_value, increment_by FROM "oc_account_terms_id_seq"': SQLSTATE[42703]: Undefined column: 7 ERROR: column "min_value" does not exist LINE 1: SELECT min_value, increment_by FROM "oc_account_terms_id_seq... ^

### Server configuration
**Operating system**:
Ubuntu 18.10 (Artful)

**Web server:**
Apache

**Database:**
Postgres 10

**PHP version:**
7.1

**ownCloud version:** (see ownCloud admin page)
10.0.7

**Updated from an older ownCloud or fresh install:**
Updated

**Where did you install ownCloud from:**
Official APT repo

If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your ownCloud installation folder

{   
    "system": {
        "updatechecker": false,
        "instanceid": "oc2cz3kt51pr",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "home.rasilon.com"
        ],
        "datadirectory": "\/var\/media\/media_big1\/ocdata\/",
        "overwrite.cli.url": "https:\/\/home.rasilon.com\/owncloud",
        "dbtype": "pgsql",
        "version": "10.0.7.2",
        "dbname": "owncloud",
        "dbhost": "localhost:5432",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "logtimezone": "UTC",
        "installed": true,
        "theme": "",
        "loglevel": 2,
        "maintenance": false
    }
}

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your ownCloud installation folder.
```
Enabled:
  - comments: 0.3.0
  - configreport: 0.1.1
  - dav: 0.3.2
  - federatedfilesharing: 0.3.1
  - federation: 0.1.0
  - files: 1.5.1
  - files_external: 0.7.1
  - files_sharing: 0.10.1
  - files_trashbin: 0.9.1
  - files_versions: 1.3.0
  - files_videoplayer: 0.9.8
  - firstrunwizard: 1.1
  - market: 0.2.4
  - notifications: 0.3.2
  - provisioning_api: 0.5.0
  - systemtags: 0.3.0
  - templateeditor: 0.3
  - updatenotification: 0.2.1
Disabled:
  - encryption
  - external
  - theme-example
  - user_external

**Are you using external storage, if yes which one:** local/smb/sftp/...
local

**Are you using encryption:** yes/no
no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
no

**Comment by phil-davis** [comment]
Like #31320 and #31334 ?

**Comment by rasilon** [comment]
Thanks.  Yes, the fix from #31334 worked.  My apologies, I didn't see it when I looked for recent bugs.
